### ISSUE TITLE
Support for progress bars?

### ISSUE THREAD

**Issue by nelson-liu** [hyperdashio/hyperdash-sdk-py#53]
Hi,
I like using tqdm to monitor my train logs, but currently it doesn't seem to work very well with hyperdash (totally understandable, i feel like this is an edge case). Is there anything i can tweak on my progress bar to get it to work, or is this more something on your side?

Thanks for this project, it looks very nifty!

Screenshot of what the progrss bar looks like on mobile:
![20731367_1889393554409486_1585511841_n](https://user-images.githubusercontent.com/7272031/29042118-84c38a18-7b6a-11e7-917f-5803b946d319.jpg)

**Comment by andrewschreiber** [comment]
Hey thanks for the bug report! Terminal loading bars tend to work via deleting the previous text via the backspace character, then printing new text.

Hyperdash currently just ignores the backspace characters and logs the new text. We'll give some thought to a smarter solution!

**Comment by richardartoul** [comment]
Yep to get this to work we'll need to make the mobile app interpret the output the same way a terminal emulator does. Will take a stab at it and see what we can do.
